### ISSUE TITLE
Balance request for an asset with zero balance: JSON parse exception happens

### ISSUE THREAD

**Issue by atemerev** [wavesplatform/WavesJ#2]
When I try to use node.getBalance(address, assetId) method, if there is no balance for some asset id, JSON parse exception occurs (as underlying HTTP API returns 404). It should return 0 instead, or at least throw more appropriate exception.

**Comment by petermz** [comment]
Can't reproduce the error. My code:
```
public static void main(String[] args) throws IOException, URISyntaxException {
    Node mainnet = new Node("https://nodes.wavesnodes.com");
    String address = "3PAHLXDbPi7h6yTCXxtHYT6Ekb6k2WF9PFD";
    System.out.println(mainnet.getBalance(address, "DDSXiNYzGuPu8sbuZYehnEouYDsBxuMoMRzrFuU8D7ri"));
    System.out.println(mainnet.getBalance(address, "9Z9J1JrD7zSKcQL61KLLftQNFKm9yiZH4UHzpP5YgBg1"));
}
```
Output:
```
0
100000000
```
What does `curl -D - https://nodes.wavesnodes.com/assets/balance/3PAHLXDbPi7h6yTCXxtHYT6Ekb6k2WF9PFD/DDSXiNYzGuPu8sbuZYehnEouYDsBxuMoMRzrFuU8D7ri` return for you?

**Comment by atemerev** [comment]
I figured out that this happened only on version 0.2; on 0.3, it works correctly (except when empty asset ID is used for Waves, but there is a separate method for Waves balance). So I think it is resolved.

**Comment by petermz** [comment]
My bad, wrong issue# above

Fixed that Waves balance thing in commit 40f192b7390775ed03badb5dfe4af64d3dd89c52
